src/v2/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -114,6 +114,9 @@ export const CollectionApp: React.FC<CollectionAppProps> = props => {
             )}
             <Box>
               <ArtworkFilterContextProvider
+                // It's possible to navigate between collections directly.
+                // Force a re-render when the pathname changes.
+                key={location.pathname}
```

Comment:
cool, does this work for the collection pages? Maybe we should add this to the other instances? I'm pretty agnostic (front) on this, I just had that `routeTracking` type approach in mind is all.

Reply:
Eh, no you're right — pushed a more general solution.